sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/QueryContainOperator.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// <see href="https://docs.microsoft.com/en-us/azure/digital-twins/reference-query-operators#contains-operators">Contains operators</see> in the ADT Query language.
+    /// </summary>
+    public enum QueryContainOperator
```

Comment:
The name is a bit misleading as it mentions "Contains" Operator, especially given the fact it's public. I don't have a better suggestion :D

Reply:
I'd at least make it `Contains` to match the actual operator